tempesta_fw/t/functional/sessions/test_sticky_sess_stress.py

```diff
@@ -34,11 +34,19 @@ def create_servers(self):
 
     def assert_servers(self):
         self.servers_get_stats()
-        # Wrk does not treat 302 responses as errors, but 302 response means
-        # that message was not forwarded to server.
-        expected_err = int(tf_cfg.cfg.get('General', 'concurrent_connections'))
-        exp_min = self.wrk.requests -  expected_err - 1
-        exp_max = self.wrk.requests
+        # Negative allowance: this means some requests are missed by the server.
+        # This happens because some (at most one) requests in each connection
+        # are sent without a session cookie and replied 302 by Tempesta without
+        # any forwarding, which is considered a "success" by wrk.
+        # So, [0; concurrent_connections] requests will be missed by the backend.
+        # This allowance is specific to the session stress tests.
+        exp_min = self.wrk.requests - self.wrk.connections
+        # Positive allowance: this means some responses are missed by the client.
+        # It is believed (nobody actually checked though...) that wrk does not
+        # wait for responses to last requests in each connection before closing it
+        # and does not account for those requests.
+        # So, [0; concurrent_connections] requests will be missed by the client.
+        exp_max = self.wrk.requests + self.wrk.connections
```

Comment:
Positive allowance is common across all stress test cases using wrk, so it should be documented in `testers.stress` whis is a base class for all such tests.